include/xplat/Starboard/SmartTypes.h

```diff
@@ -18,6 +18,11 @@
 
 #include "Starboard/TypeTraits.h"
 
+// IwFree symbol must already be declared when not building on Windows
+#ifndef _WIN32
```

Comment:
Does it hurt to just always include this header? IWMalloc.h should be #pragma once'd so this shouldn't matter.